code/frameworks/nextjs/src/css/webpack.ts

```diff
@@ -0,0 +1,108 @@
+import { NextConfig } from 'next';
+import { getCssModuleLocalIdent } from 'next/dist/build/webpack/config/blocks/css/loaders/getCssModuleLocalIdent';
+import { cssFileResolve } from 'next/dist/build/webpack/config/blocks/css/loaders/file-resolve';
+import { Configuration as WebpackConfig } from 'webpack';
+import semver from 'semver';
+import { scopedResolve } from '../utils';
+
+export const configureCss = (baseConfig: WebpackConfig, nextConfig: NextConfig): void => {
```

Comment:
just a heads up: when developing this, this file had a large amount of code churn because nextjs's css configuration is very specific. Originally, I thought a lot of the configuration was extraneous for the purposes of storybook, but many people depend on the various things that the nextjs css config supports. It is worth keeping the nextjs css config files bookmarked.